src/distances.jl

```diff
@@ -24,53 +32,76 @@ julia> Ellipsoidal([1.0,0.5], [π/2])
 julia> Ellipsoidal([1.0,0.5,0.5], [π/2,0.0,0.0])
 ```
 """
+
+# Right-hand rule; motion defined looking to the negative direction of the axis
```

Comment:
Perhaps it would be a good idea to add a more detailed docstring for this struct explaining the meaning of each of the fields. That can help people interested in adding new conventions, though I thin you already covered it all :)

Reply:
Check out what you think about the change